### PR TITLE
rbsa roa added

### PR DESCRIPTION
**Pull request by berkserbet**
### Why:
* rbsa roa implementation

### What:
* applies RoA to rbsa normalized loadshapes. Needed roa file can be found in S3.

### Checklist
- [x] Finished My Changes
- [ ] Automated Unit Tests

### CODE REVIEW

**Comment by berkserbet** [comment]
This does not contain plotting for roa output. I can add that once other branches are merged.

File: pipelines/rbsa/tasks/apply_roa.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        locations = self.df.target.unique()
+        daytypes = self.df.daytype.unique()
+
+        components_df = pd.DataFrame(columns=['target','daytype','time'] + list(self.roa.index))
+
+        for location in locations:
+            for daytype in daytypes:
+                day_df = self.df.loc[(self.df.target == location) & (self.df.daytype == daytype)]
+                day_matrix = day_df[ROA_columns].values
+                component_matrix = np.matmul(day_matrix,roa_matrix.T)
```
Comment (jongoncalves) [inline pipelines/rbsa/tasks/apply_roa.py]: space after comma

Comment (berkserbet) [reply pipelines/rbsa/tasks/apply_roa.py]: fixed

**Comment by berkserbet** [comment]
@jongoncalves Added ROA plotting to this function. Will merge after showing Dave and Joe plots.

**Pull request merged**